api/services/gameService.js

```diff
@@ -34,43 +34,33 @@ async function fetchSpectatorUsernames(gameId) {
 
 // Used to create fully populated game
 function tempUser(usr, points) {
```

Comment:
I don't like the names tempUser or tempGame, but I'm not sure what to call them. For one I think these methods will be cleaner if they return a new object rather than using `this` for the class-like assignment approach. With that in mind, it probably makes sense for the function names to be verbs describing the fact that they format data, rather than object-like names. How about `formatPlayerData()` and `formatGameData()`?

Reply:
I'm open to other suggestions. I'd like to make this more clear.

Reply:
I think those work, the only others i can really think of might be addUserDetails and addGameDetails